### ISSUE TITLE
Info on fail2ban section

### ISSUE THREAD

**Issue by docop** [krathalan/systemd-sandboxing#3]
Hi
 i would like to know about the fail2ban section.. was it really usefull to add this one ? As if you get into a system, it's too late.. And the protection may not be needed as fail2ban by itself should block and ban connection directly ?

thanks on precision..

**Comment by krathalan** [comment]
Correct, it is not very necessary. I made a lot of these profiles simply because they're long-running daemons. Frankly I do need to clean up this repo soon but I have been very busy with work.
